sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -4006,13 +4004,37 @@ var setParentClass = function(child, parent) {
                 this.m_var1, this.m_var2, this.m_var3);
 
             this.m_ret = this.doExpand(ctx, args);
-            if (this.doExpandAlternative !== undefined) {
-                this.m_ret_alternative = this.doExpandAlternative(ctx, args);
-            }
             this.m_body = ctx.getStatements();
         }
     };
 
+    /**
+     * @constructor
+     * @extends {glsBuiltinPrecisionTests.Func}
+     * @param {glsBuiltinPrecisionTests.Signature} Sig_ template <typename Sig_>
+     */
+    glsBuiltinPrecisionTests.Alternatives = function(Sig_) {
+         glsBuiltinPrecisionTests.Func.call(this, Sig_);
```

Comment:
Not sure, but the dEQP files are all problematic. They use tabs instead of spaces. I'm not sure it is a good idea to reformat them because doing so will lose "git blame" information.